src/types/rule-promotions.ts

```diff
@@ -53,7 +53,7 @@ export interface Condition {
 export interface RulePromotionBase {
   type: 'rule_promotion'
   name: string
-  description: string
+  description?: string | undefined
```

Comment:
I thought just 
```suggestion
  description?: string 
```
would suffice?